### DISCUSSION TITLE
Feature request. Click and drag action for all buttons and drag directions. [Moved to Issue #350]

### DISCUSSION THREAD

**Discussion by NoUsername10** [noah-nuebling/mac-mouse-fix#338] [Other]
Hi!
Fantastic software, love it!
As the title says, please delete this if it's been asked or requested before, could not find it when searching.

Add actions for click (any button) and drag (in all directions, up, down, left, right) for all buttons.

Hope you have a nice day.

**Comment by noah-nuebling** [comment]
Hi @NoUsername10, thanks so much for the feedback and for your kind words! I'm really glad you're loving the app! 

This is a really cool feature that I've also been thinking about and even put a little bit of work into! It's not coming super soon but I would really love to add it.

I've created Issue #350 to track your idea. Please refer to that for further discussion. Thanks! And let me know if you have more questions or ideas!

**Reply by NoUsername10** [reply]
Hello @noah-nuebling.
I don't know this person teleV2 and i don't know if this is an english language difficulty thing or scamming.
So it's not me in a different form, perhaps this person also had this request but want to use the feature in a different way.

**Reply by teleV2** [reply]
oh, god! I apologize I am new to github and this is exactly the same what I wrote a while ago so I misunderstood that this was mine. maybe I didn't post mine at that time. I will delete my comment above. Instead, I give a thumb up here. Sorry again

**Reply by NoUsername10** [reply]
@teleV2 
No worries at all, all is good!

**Reply by noah-nuebling** [reply]
Hahaha that's hilarious! Thanks for the kind words @teleV2! :)

**Comment by NoUsername10** [comment]
Hi @noah-nuebling.
I used Options+ but it doesn't have smooth scrolling so your app is a daily driver!
Happy to hear you thought about this feature before!

Hopefully you get time to implement it in the future, will track #350 for future updates :-) 
Have a great day!
